subprojects/platform-play/src/test/groovy/org/gradle/play/plugins/PlayTestPluginTest.groovy

```diff
@@ -57,6 +58,7 @@ class PlayTestPluginTest extends Specification {
             taskName(_, _) >> { String verb, String object -> "${verb}SomeBinary${object.capitalize()}"}
             taskName(_) >> { String verb -> "${verb}SomeBinary"}
         }
+        _ * binary.jarFile >> jarFile
```

Comment:
Why is this needed now? Also, let's use explicit numbers of repetitions instead of `_` to avoid leaving unused code in the tests.

Reply:
`SimpleFileCollection` happily took the `null` produced by the `binary` `Mock` from `getJarFile()`.  However, now the `ImmutableFileCollection` throws an NPE, so we actually need to stub out the value.  I've updated it to be the right number for this call.

Reply:
Thanks. Can we also add `0 * _`?

Reply:
I added  `0 * _` in 5a90200e4d1, but this whole test class is pretty awful.